internal/versions/versions.go

```diff
@@ -266,6 +269,19 @@ func (h *handler) validateVersions() error {
 		if osImage.Version == nil {
 			return errors.Errorf(fmt.Sprintf(missingValueTemplate, "Version"))
 		}
+
+		missingValueTemplate = "Missing value in ReleaseImage for '%s' field"
+		for _, release := range h.releaseImages {
+			if release.URL == nil {
+				return errors.Errorf(fmt.Sprintf(missingValueTemplate, "URL"))
```

Comment:
Sorry for not noticing this earlier, but do you think we should use the `json` field name instead of Go's? 

Considering this is a user-facing message, we may want to tell them exactly what's missing in the json config.